sponsorship/src/test/resources/application.yaml

```diff
@@ -0,0 +1,7 @@
+spring:
+  liquibase.change-log: classpath:db/changelog/changelog-master.xml

```

Comment:
Why do you define this again?

Reply:
It is a configuration for tests, DB is started in AbstractContainerBaseTest with PostgreSQLContainer

Reply:
But what is the difference between default config and test profile? They are both applied, this is why I'm asking.

Reply:
Yep, it is not needed. I'll remove it